samples/charts/pie-chart/explosion/src/PieChartExplosion.ts

```diff
@@ -16,6 +18,8 @@ export class PieChartExplosion {
         this.chart = document.getElementById('chart') as IgcPieChartComponent;
         this.chart.dataSource = this.getData();
         this.chart.sliceClick = this.onSliceClick;
+        this.chart.legend = document.getElementById('legend') as IgcItemLegendComponent;
+
```

Comment:
this sample is missing code for exploding slices

Reply:
slice is exploded through html : exploded-slices="1, 2"